assets/app/view/game/spreadsheet.rb

```diff
@@ -20,6 +20,8 @@ def render
         @spreadsheet_sort_order = Lib::Storage['spreadsheet_sort_order']
 
         @players = @game.players.reject(&:bankrupt)
+        @hide_ipo = @game.all_corporations.reject(&:minor?).all?(&:always_market_price)
+        @show_corporation_size = @game.all_corporations.any?(&@game.method(:show_corporation_size?))
```

Comment:
what is this?

Reply:
It's checking if there is some corp where those columns will be used. If not, this variable is checked later to hide the column

Reply:
rewrite it any? { |c| c.repsondsto}